assignments/prototypes.js

```diff
@@ -32,79 +59,99 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+
+
+
+function Humanoid(attributes) {
+    CharacterStats.call(this, object)
```

Comment:
I got `object` is undefined when I ran your code. I fixed it by passing `attributes` instead.